### DISCUSSION TITLE
Setting Slash-Command permissions

### DISCUSSION THREAD

**Discussion by FGBxRamel** [nextcord/nextcord#408] [Q&A] [answered]
So you are able (or not rn https://github.com/nextcord/nextcord/issues/373, but let's ignore that) to set `default_permission=False` to prevent that everyone can use the command. But how do I set who can use it? Can't seem to find something in the docs and the code.

**Comment by TAG-Epic** [comment, marked as answer]
Wait for discord to release perms v2. Sorry i dont want to waste time on something pending removal

**Reply by ooliver1** [reply]
Following up to this, the current slash perms is role based and very half put together but is getting replaced hopefully soon

**Reply by FGBxRamel** [reply]
Thank you very much, have a great day^^
